README.md

```diff
@@ -226,9 +226,25 @@ New statuses are up to the commissioners' discretion until the league votes on t
 ### Toilet Bowl ###
 - Seeding will be determined by the end of the regular season standings for the 4 teams not in the Championship Bracket
 - The consolation bracket has no bearing on the draft order
-- The winner of the Toilet Bowl must wear the toilet seat at the draft and pays for the championship trophy plaque
-
-<br>
+- The winner of the Toilet Bowl must wear the toilet seat at the draft and pay for the championship trophy plaque
+
+### Golf ###
+- Takes place Championship week (Week 17)
+- The lowest score wins the $20 weekly prize
+**Rules**
+- If a player doesn't play any snaps, then the owner is disqualified
+- If a player doesn't score any points but plays a snap, then they receive 20 points
+- A player is classified as scoring points if, at any point during the game, their total points change (not zero)
```

Comment:
maybe add examples here for what counts and what doesn't. (e.g., a fumble for -2 would count, 2 carries one for 1, one for -1 yards would count, 0/2 targets no points would not, etc.)

Reply:
maybe like "Player must do at least one point scoring action (positive or negative points)

Reply:
I like this. I added some clarifying examples